### DISCUSSION TITLE
Blueman doesn't want to search for any devices

### DISCUSSION THREAD

**Discussion by atomi19** [blueman-project/blueman#2580] [Q&A]
blueman: 2.4
BlueZ:
Distribution: Linux Mint
Desktop environment: Cinnamon

Hi, i got some problem with blueman-manager, so it doesn't want search for any devices, only after reboot system, then maybe it will work, but not always. All buttons are not available(except exit)

![blueman_error](https://github.com/user-attachments/assets/aba417e2-d092-44b2-b392-4c895a0fd22a)

**Comment by romychs** [comment]
Same issue with Blueman: v2.3.5 and Linux Mint 21.3 Virginia, Cinnamon
Button  [Search] available, but no devices found.
USB-Bluetooth adapter
ID 0bda:8771 Realtek Semiconductor Corp. Bluetooth Radio
Under windows, device work normal.

**Comment by ghost** [comment]
Gleiches Problemm mit Mint 22.

**Comment by cschramm** [comment]
Anything in `bluetoothctl list`? Otherwise, your system does not have any adapter set-up.

**Reply by romychs** [reply]
You right. It is something in system. In my case only device byself in list.
    root@wst-pc:~# bluetoothctl list
    Controller 8C:88:0B:69:47:2C wst-pc [default]

**Reply by atomi19** [reply]
Hi, sometimes when i type **bluetoothctl list**, there is no output, but sometimes also it returns me that(and when i am getting that i can search for devices and all buttons are available):
Controller A4:34:D9:E6:3A:9C ProBook [default]

![blueman_success](https://github.com/user-attachments/assets/34009644-b465-444b-a6bc-04074c3e0595)

**Reply by cschramm** [reply]
If BlueZ does not have any working adapter, blueman cannot do anything for you (except unblocking a software killswitch that automagically connects an adapter). An adapter showing up from time to time is weird. I'd check `dmesg` for anything related and possibly `bluetooth.service`'s log.
